apps/server/src/modules/tldraw/job/tldraw-files.console.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { createMock } from '@golevelup/ts-jest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { LegacyLogger } from '@src/core/logger';
+import { TldrawDeleteFilesUc } from '../uc';
+import { TldrawFilesConsole } from './tldraw-files.console';
+
+describe('TldrawFilesConsole', () => {
+	let console: TldrawFilesConsole;
+	let deleteFilesUc: TldrawDeleteFilesUc;
+
+	beforeAll(async () => {
+		const module: TestingModule = await Test.createTestingModule({
+			providers: [
+				TldrawFilesConsole,
+				{
+					provide: TldrawDeleteFilesUc,
+					useValue: createMock<TldrawDeleteFilesUc>(),
+				},
+				{
+					provide: LegacyLogger,
+					useValue: createMock<LegacyLogger>(),
+				},
+			],
+		}).compile();
+
+		console = module.get(TldrawFilesConsole);
+		deleteFilesUc = module.get(TldrawDeleteFilesUc);
+
+		// Set fake system time. Otherwise, dates constructed in the test and the
+		// console can differ because of the short time elapsing between the calls.
+		jest.useFakeTimers();
+		jest.setSystemTime(new Date(2022, 1, 22));
+	});
+
+	it('should be defined', () => {
+		expect(console).toBeDefined();
+	});
+
+	describe('deleteUnusedFiles', () => {
+		it('should call UC with threshold date', async () => {
```

Comment:
I do not see any .console.spec.ts file written like you described here. All of them use mocks.
I am sorry but integration tests here involving saving images to tldraw, storing this document in db and then using RabbitMQ to mark them as deleted might be quite hard. I feel like unit tests here are enough, they are covering all the cases in my opinion.
But if you really do feel like we need this integration test then I'd ask you to create separate ticket for that, as for me it would be harder and more time consuming than doing cronjob implementation itself.